iree/samples/ops/mhlo-dot.mlir

```diff
@@ -0,0 +1,5 @@
+func @dot(%lhs: tensor<32x1024xf32>, %rhs: tensor<1024x64xf32>) attributes { iree.module.export } {
+  %res = "mhlo.dot"(%lhs, %rhs) : (tensor<32x1024xf32>, tensor<1024x64xf32>) -> tensor<32x64xf32>
+  check.expect_almost_eq_const(%res, dense<409.596> : tensor<32x64xf32>) : tensor<32x64xf32>
```

Comment:
Do we expect to let the check framework get involved in the example?

Reply:
Oops. Copy-pasta..